entity-service/build.gradle.kts

```diff
@@ -19,7 +19,7 @@ dependencies {
   implementation("org.hypertrace.core.grpcutils:grpc-server-utils:0.6.1")
   implementation("org.hypertrace.core.grpcutils:grpc-client-utils:0.6.1")
   implementation("org.hypertrace.core.serviceframework:platform-service-framework:0.1.28")
-  implementation("org.hypertrace.core.documentstore:document-store:0.6.0")
+  implementation("org.hypertrace.core.documentstore:document-store:0.6.1")
```

Comment:
Does the service container use doc store directly? If yes, change is good as is. If no, we can remove the dependency entirely.